src/core/api.h

```diff
@@ -5,6 +5,13 @@
 
 --*/
 
+//
+// Private flag of the QUIC_CONNECTION_SHUTDOWN_FLAGS enum used for indicating
+// the type of error code being passed in. Not exposed to apps because they
```

Comment:
nti:
```suggestion
// a QUIC_STATUS is being passed (and not a QUIC_ERROR). 
// Not exposed to apps because they
```

Reply:
The "not a QUIC_ERROR" isn't actually correct either 😄 I think I'd rather just leave it as is for now and not write a massive comment explaining the various types of errors in MsQuic